ocaml/quicktest/quicktest_max_vdi_size.ml

```diff
@@ -1,19 +1,12 @@
-
-(** From XenServer 7.5 Configuration Limits:
-    https://docs.citrix.com/content/dam/docs/en-us/xenserver/current-release/downloads/xenserver-config-limits.pdf
-    Using (2TiB - 4GiB), instead of the (2TB - 4GB) limit defined in the above
-    document, does not work, we cannot create a VDI of that size. *)
-let max_vdi_size = Sizes.((2L ** tb) -* (4L ** gb))
-
 let with_max_vdi rpc session_id sr f =
-  Qt.VDI.with_new rpc session_id ~virtual_size:max_vdi_size sr
+  Qt.VDI.with_new rpc session_id ~virtual_size:Constants.max_vhd_size sr
```

Comment:
Again - what is too large: the VHD or the VDI?

Reply:
True again. We should rename the function to `with_max_vhd`.